webrtc.html

```diff
@@ -5815,7 +5815,9 @@ <h2>Methods</h2>
                     <li>
                       <p>Determine if negotiation is needed to transmit
                       <var>withTrack</var> in place of the sender's existing
-                      track. Negotiation is not needed if <var>withTrack</var>
+                      track. The details of this decision is left to
+                      implementations with exception that
```

Comment:
I don't think this is  right. They are defined by SDP offer / answer and JSEP. Basically if what is being sent would be outside the currently negotiated SDP offer/answer envelope, then negotiation is needed *before* it can be sent.

Reply:
Agree with Cullen. Given the divergence in replaceTrack implementations, more specific guidance is probably preferable to endorsing "implementation specific" behavior.  Can we discuss this at the January Virtual Interim?